diff --git a/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs b/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
index 6000445..217d9f6 100644
--- a/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
+++ b/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
@@ -382,7 +382,7 @@ var ajvOrig = (additionalOptions = {}) => {
 
     ajv.addMetaSchema(metaSchema);
     // eslint-disable-next-line no-underscore-dangle -- part of the API
-    ajv._opts.defaultMeta = metaSchema.id;
+    if (ajv._opts) { ajv._opts.defaultMeta = metaSchema.id; }
 
     return ajv;
 };
diff --git a/node_modules/@eslint/eslintrc/dist/eslintrc.cjs b/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
index 1b76091..f352d0e 100644
--- a/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
+++ b/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
@@ -1615,7 +1615,7 @@ var ajvOrig = (additionalOptions = {}) => {
 
     ajv.addMetaSchema(metaSchema);
     // eslint-disable-next-line no-underscore-dangle -- part of the API
-    ajv._opts.defaultMeta = metaSchema.id;
+    if (ajv._opts) { ajv._opts.defaultMeta = metaSchema.id; }
 
     return ajv;
 };
