diff --git a/node_modules/@react-native/gradle-plugin/build.gradle.kts b/node_modules/@react-native/gradle-plugin/build.gradle.kts
index 9346508..772a4e4 100644
--- a/node_modules/@react-native/gradle-plugin/build.gradle.kts
+++ b/node_modules/@react-native/gradle-plugin/build.gradle.kts
@@ -5,7 +5,6 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import org.gradle.api.internal.classpath.ModuleRegistry
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
@@ -48,13 +47,7 @@ dependencies {
   testImplementation(libs.junit)
 
   testRuntimeOnly(
-      files(
-          gradle.sharedServices
-              .getByType(ModuleRegistry::class)
-              .getModule("gradle-tooling-api-builders")
-              .classpath
-              .asFiles
-              .first()))
+      files())
 }
 
 // We intentionally don't build for Java 17 as users will see a cryptic bytecode version
