diff --git a/node_modules/@react-native/gradle-plugin/build.gradle.kts b/node_modules/@react-native/gradle-plugin/build.gradle.kts
index 1000000..2000000 100644
--- a/node_modules/@react-native/gradle-plugin/build.gradle.kts
+++ b/node_modules/@react-native/gradle-plugin/build.gradle.kts
@@ -1,5 +1,4 @@
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
-import org.gradle.configurationcache.extensions.serviceOf
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
