diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.11/executionHistory/executionHistory.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.11/executionHistory/executionHistory.bin
new file mode 100644
index 0000000..41b92a7
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.11/executionHistory/executionHistory.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.11/executionHistory/executionHistory.lock b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.11/executionHistory/executionHistory.lock
new file mode 100644
index 0000000..13cf77c
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.11/executionHistory/executionHistory.lock differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.13/executionHistory/executionHistory.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.13/executionHistory/executionHistory.bin
new file mode 100644
index 0000000..09be38d
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.13/executionHistory/executionHistory.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.13/executionHistory/executionHistory.lock b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.13/executionHistory/executionHistory.lock
new file mode 100644
index 0000000..550d47d
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.13/executionHistory/executionHistory.lock differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.bin
new file mode 100644
index 0000000..8f631aa
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.lock b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.lock
new file mode 100644
index 0000000..a3d7086
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.lock differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.3/executionHistory/executionHistory.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.3/executionHistory/executionHistory.bin
new file mode 100644
index 0000000..fd1b9e3
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.3/executionHistory/executionHistory.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.3/executionHistory/executionHistory.lock b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.3/executionHistory/executionHistory.lock
new file mode 100644
index 0000000..cef7e7c
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.3/executionHistory/executionHistory.lock differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/buildOutputCleanup.lock b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/buildOutputCleanup.lock
new file mode 100644
index 0000000..61ffac1
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/buildOutputCleanup.lock differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/cache.properties b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
index 0000000..2248e7e
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/cache.properties
@@ -0,0 +1,2 @@
+#Thu Feb 19 02:28:57 PST 2026
+gradle.version=8.3
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/outputFiles.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/outputFiles.bin
new file mode 100644
index 0000000..1055223
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/outputFiles.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/file-system.probe b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/file-system.probe
new file mode 100644
index 0000000..c4d7342
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/file-system.probe differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/dirty-sources.txt b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/dirty-sources.txt
new file mode 100644
index 0000000..a5e0fc8
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/dirty-sources.txt
@@ -0,0 +1,11 @@
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\android\MavenPublicationExtension.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\gradle\ExpoModuleExtension.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\Warnings.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\withAutolinkingPlugin\kotlin\expo\modules\plugin\AutolinkingIntegrationImpl.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\gradle\ExpoGradleHelperExtension.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\ProjectConfiguration.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\ExpoModulesGradlePlugin.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\Version.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\android\AndroidLibraryExtension.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\AutolinkingIntegration.kt
+C:\Users\Administrator\Downloads\virtucam\node_modules\expo-modules-core\expo-module-gradle-plugin\src\main\kotlin\expo\modules\plugin\ExtraPropertiesExtension.kt
\ No newline at end of file
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/local-state/build-history.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/local-state/build-history.bin
new file mode 100644
index 0000000..e50ebd5
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/local-state/build-history.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/libs/expo-module-gradle-plugin.jar b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/libs/expo-module-gradle-plugin.jar
new file mode 100644
index 0000000..2315a72
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/libs/expo-module-gradle-plugin.jar differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/pluginDescriptors/expo-module-gradle-plugin.properties b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/pluginDescriptors/expo-module-gradle-plugin.properties
new file mode 100644
index 0000000..08df140
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/pluginDescriptors/expo-module-gradle-plugin.properties
@@ -0,0 +1 @@
+implementation-class=expo.modules.plugin.ExpoModulesGradlePlugin
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/resources/main/META-INF/gradle-plugins/expo-module-gradle-plugin.properties b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/resources/main/META-INF/gradle-plugins/expo-module-gradle-plugin.properties
new file mode 100644
index 0000000..08df140
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/resources/main/META-INF/gradle-plugins/expo-module-gradle-plugin.properties
@@ -0,0 +1 @@
+implementation-class=expo.modules.plugin.ExpoModulesGradlePlugin
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/tmp/jar/MANIFEST.MF b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/tmp/jar/MANIFEST.MF
new file mode 100644
index 0000000..59499bc
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/tmp/jar/MANIFEST.MF
@@ -0,0 +1,2 @@
+Manifest-Version: 1.0
+
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ExpoModulesGradlePlugin.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ExpoModulesGradlePlugin.kt
index 71d54fc..c2d0102 100644
--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ExpoModulesGradlePlugin.kt
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ExpoModulesGradlePlugin.kt
@@ -7,7 +7,6 @@ import expo.modules.plugin.gradle.ExpoGradleHelperExtension
 import expo.modules.plugin.gradle.ExpoModuleExtension
 import org.gradle.api.Plugin
 import org.gradle.api.Project
-import org.gradle.internal.extensions.core.extra
 
 private val lock = Any()
 
@@ -42,12 +41,12 @@ abstract class ExpoModulesGradlePlugin : Plugin<Project> {
   }
 
   private fun getKotlinVersion(project: Project): String {
-    return project.rootProject.extra.safeGet<String>("kotlinVersion")
+    return project.rootProject.extensions.extraProperties.safeGet<String>("kotlinVersion")
       ?: project.logger.warnIfNotDefined("kotlinVersion", "2.0.21")
   }
 
   private fun getKSPVersion(project: Project): String {
-    return project.rootProject.extra.safeGet<String>("kspVersion")
+    return project.rootProject.extensions.extraProperties.safeGet<String>("kspVersion")
       ?: project.logger.warnIfNotDefined("kspVersion", "2.0.21-1.0.28")
   }
 }
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ProjectConfiguration.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ProjectConfiguration.kt
index 7c22069..36411d1 100644
--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ProjectConfiguration.kt
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ProjectConfiguration.kt
@@ -15,7 +15,6 @@ import expo.modules.plugin.android.createReleasePublication
 import expo.modules.plugin.gradle.ExpoModuleExtension
 import org.gradle.api.Project
 import org.gradle.api.publish.PublishingExtension
-import org.gradle.internal.extensions.core.extra
 import java.io.File
 
 internal fun Project.applyDefaultPlugins() {
@@ -31,8 +30,8 @@ internal fun Project.applyDefaultPlugins() {
 }
 
 internal fun Project.applyKotlin(kotlinVersion: String, kspVersion: String) {
-  extra.set("kotlinVersion", kotlinVersion)
-  extra.set("kspVersion", kspVersion)
+  extensions.extraProperties.set("kotlinVersion", kotlinVersion)
+  extensions.extraProperties.set("kspVersion", kspVersion)
 
   project.dependencies.add("implementation", "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion")
 }
@@ -50,11 +49,11 @@ internal fun Project.applyDefaultDependencies() {
 internal fun Project.applyDefaultAndroidSdkVersions() {
   with(androidLibraryExtension()) {
     applySDKVersions(
-      compileSdk = rootProject.extra.safeGet("compileSdkVersion")
+      compileSdk = rootProject.extensions.extraProperties.safeGet("compileSdkVersion")
         ?: logger.warnIfNotDefined("compileSdkVersion", 36),
-      minSdk = rootProject.extra.safeGet("minSdkVersion")
+      minSdk = rootProject.extensions.extraProperties.safeGet("minSdkVersion")
         ?: logger.warnIfNotDefined("minSdkVersion", 24),
-      targetSdk = rootProject.extra.safeGet("targetSdkVersion")
+      targetSdk = rootProject.extensions.extraProperties.safeGet("targetSdkVersion")
         ?: logger.warnIfNotDefined("targetSdkVersion", 36)
     )
     applyLinterOptions()
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/gradle/ExpoModuleExtension.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/gradle/ExpoModuleExtension.kt
index 896cc50..cde7995 100644
--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/gradle/ExpoModuleExtension.kt
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/gradle/ExpoModuleExtension.kt
@@ -7,7 +7,6 @@ import java.io.File
 import java.util.Properties
 import expo.modules.plugin.Version
 import expo.modules.plugin.safeGet
-import org.gradle.internal.extensions.core.extra
 
 /**
  * An user-facing interface to interact with the `ExpoGradleHelperExtension`.
@@ -31,7 +30,7 @@ open class ExpoModuleExtension(val project: Project) {
     get() = gradleHelper.getReactNativeVersion(project)
 
   fun safeExtGet(name: String, default: Any): Any {
-    return project.rootProject.extra.safeGet<Any>(name) ?: default
+    return project.rootProject.extensions.extraProperties.safeGet<Any>(name) ?: default
   }
 
   fun getExpoDependency(name: String): Any {
